src/Compilers/Server/CommandLine/VBCSCompilerCommandLine.shproj

```diff
@@ -0,0 +1,16 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
+<Project>
+  <PropertyGroup Label="Globals">
+    <ProjectGuid>d8ef0777-9d65-4849-a7d6-ac81e58e2317</ProjectGuid>
+    <MinimumVisualStudioVersion>14.0</MinimumVisualStudioVersion>
+  </PropertyGroup>
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props"
+          Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)\CodeSharing\Microsoft.CodeSharing.Common.Default.props"
```

Comment:
I didn't understand what these projects are. Where did they come from? Does VS need to be installed in order to build these?

Reply:
I don't know what they are either but it's boiler plate in every .shproj hence included here. Perhaps @rainersigwald, or @baronfel know the reason for them existing

Reply:
No idea, sorry. My assumption would be that it lights up VS functionality but isn't required at build time.